### PR TITLE
refactor(analytics): update content open identifiers

### PR DESCRIPTION
**Pull request by dskuza**
## Summary

Updates ContentOpen event identifiers when:
1. Opening an item from the user's list
2. Opening an item in web view from the reader
3. Opening a link by tapping on a link in the reader

## References 

IN-1343

## Test Steps

Looking at the above summary, when testing step:
1. The identifier should be `saves.card.open`
2. The identifier should be `reader.view-original`
3. The identifier should be `reader.view-external`

## PR Checklist:
- [x] Added Unit / UI tests
- [x] Self Review (review, clean up, documentation, run tests)
- [x] Basic Self QA

### CODE REVIEW

File: PocketKit/Sources/Analytics/AppEvents/Reader.swift [hunk 1]
```diff
@@ -76,7 +76,17 @@ public extension Events.Reader {
             contentEntity: ContentEntity(url: url),
             uiEntity: UiEntity(
                 .button,
-                identifier: "reader.view_original"
+                identifier: "reader.view-original"
+            )
+        )
+    }
+
+    static func openExternalLink(url: URL) -> ContentOpen {
```
Comment (bassrock) [inline PocketKit/Sources/Analytics/AppEvents/Reader.swift:84]: @dskuza can you define an external link open in the description. im unclear if that means opening in web view? or said differently... how does one open an external link in reader?

**Review by bassrock — APPROVED** [review]
nit, but not a blocker comment.

**Comment by bassrock** [comment]
@dskuza i see UI Test 1 failing, can you check that since @cyndichin fixed it earlier today.

File: PocketKit/Sources/Analytics/AppEvents/Reader.swift (hunk 1 above)
Comment (dskuza) [reply PocketKit/Sources/Analytics/AppEvents/Reader.swift:84]: yeah, i'll go back and leave a comment, but…

an external link is a link that exists within the reader view (not a webview), usually in a text component. when a link is tapped in the reader, it is an "external link". alternatively, if you long-press a link, there is an "open" action that is also the same thing.

**Comment by dskuza** [comment]
> @dskuza i see UI Test 1 failing, can you check that since @cyndichin fixed it earlier today.

i definitely broke it in the pull request; looking now

**Review by bassrock — APPROVED** [review]

**Pull request merged**